ci/utils.py

```diff
@@ -168,3 +173,84 @@ def upload_bundle(
     source = f"https://github.com/{repo_name}/releases/download/{release_tag}/{bundle_zip_filename}"
 
     return source
+
+
+def find_bundle_file(root_dir: str, file: str, suffix=("json", "yaml", "yml")):
+    # find bundle file with possible suffix
+    file_name = None
+    for name in suffix:
+        full_name = f"{file}.{name}"
+        if full_name in os.listdir(root_dir):
+            file_name = full_name
+
+    return file_name
+
+
+def produce_dataset(dataset_dir, dataset_size, input_shape):
+    for s in range(dataset_size):
+        test_image = np.random.randint(low=0, high=2, size=input_shape).astype(np.int8)
+        test_label = np.random.randint(low=0, high=2, size=input_shape).astype(np.int8)
+        image_filename = os.path.join(dataset_dir, f"image_{s}.nii.gz")
+        label_filename = os.path.join(dataset_dir, f"label_{s}.nii.gz")
+        nib.save(nib.Nifti1Image(test_image, np.eye(4)), image_filename)
+        nib.save(nib.Nifti1Image(test_label, np.eye(4)), label_filename)
+    print(f"produced fake dataset in {dataset_dir} for test.")
+
+
+class TestBundleConfigs:
+    """
+    A standard test bundle config class
+    """
+
+    def __init__(
+        self,
+        bundle_root,
+        dataset_size=16,
+        input_shape=(224, 224, 224),
+        check_determinism=False,
+        check_multi_gpu_train=False,
+        check_multi_gpu_evaluate=False,
+        train_override=None,
+        inference_override=None,
+    ):
+        self.bundle_root = bundle_root
+        self.input_shape = input_shape
+        self.dataset_size = dataset_size
+        self.check_determinism = check_determinism
+        self.check_multi_gpu_train = check_multi_gpu_train
+        self.check_multi_gpu_evaluate = check_multi_gpu_evaluate
+        self.dataset_dir = tempfile.mkdtemp()
+        self.train_override = train_override
+        self.inference_override = inference_override
+        monai.utils.set_determinism(seed=123)
+        produce_dataset(self.dataset_dir, self.dataset_size, self.input_shape)
+
+    def assign_override_default_info(self, override_dict):
+        override_dict["bundle_root"] = self.bundle_root
+        override_dict["dataset_dir"] = self.dataset_dir
+        return override_dict
+
+    def test_train_config(self):
```

Comment:
Hi @Nic-Ma @wyli @KumoLiu , could you please help to give some suggestions on this PR? If the format is fine, I will continue to add the tests on other config files, and then this PR will be ready.

Reply:
I will take a look later.

Thanks.